internal/cli/atlas/privateendpoints/gcp/interfaces/create.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interfaces
+
+import (
+	"context"
+	"strings"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+
+	"github.com/mongodb/mongocli/internal/cli"
```

Comment:
please, remove the new line and run `make fmt`

Reply:
OK, that has been done with the latest commit.